galaxy/main/management/commands/delete_collection_version.py

```diff
@@ -0,0 +1,83 @@
+from django.core.management.base import BaseCommand
+from dynaconf.contrib import django_dynaconf  # noqa
```

Comment:
@cutwater fyi the loading of settings issue I found when calling this via `django-admin shell` were resolved once I used `python manage.py shell`